Logging/.CSProject/Logger/Logger.cs

```diff
@@ -52,87 +52,87 @@ private static string GetReadableName(Type type)
         /// <summary>
         /// Creates an instance of <see cref="Logger"/> from a <see cref="Type"/>.
         /// </summary>
-        /// <param name="type">The <see cref="Type"/> to create the <see cref="Logger"/> instance for.</param>
+        /// <param name="ownerType">The <see cref="Type"/> to create a <see cref="Logger"/> instance for.</param>
         /// <param name="messagePrefix">
         /// An optional <see cref="string"/> to prefix to all messages through this logger.
         /// Useful when there are multiple types that share the same name which need to be differentiated.
         /// </param>
-        public Logger(Type type, string messagePrefix = null) : this(GetReadableName(type), messagePrefix) { }
+        public Logger(Type ownerType, string messagePrefix = null) : this(GetReadableName(ownerType), messagePrefix) { }
         /// <summary>
-        /// Creates an instance of <see cref="Logger"/> from another instance.
+        /// Creates an instance of <see cref="Logger"/> for an object instance.
         /// </summary>
-        /// <param name="instance">The instance to create the <see cref="Logger"/> instance for.</param>
+        /// <param name="owner">The instance to create a <see cref="Logger"/> instance for.</param>
         /// <param name="messagePrefix">
         /// An optional <see cref="string"/> to prefix to all messages through this logger.
         /// Useful when there are multiple instances or types that share the same name which need to be differentiated.
         /// </param>
-        public Logger(in object instance, string messagePrefix = null) : this(instance.GetType(), messagePrefix) { }
+        public Logger(in object owner, string messagePrefix = null) : this(owner.GetType(), messagePrefix) { }
 
-        private Logger(string derivedTypeName, string messagePrefix)
+        private Logger(string ownerTypeName, string messagePrefix)
         {
-            DerivedTypeName = derivedTypeName;
+            OwnerTypeName = ownerTypeName;
             MessagePrefix = messagePrefix;
         }
 
         /// <inheritdoc cref="ILogger.Debug"/>
         public void Debug(
             object message,
-            [CallerFilePath] string callerPath = "",
-            [CallerMemberName] string callerName = "",
-            [CallerLineNumber] int callerLine = 0
+            [CallerMemberName] string callerMethodName = "",
+            [CallerFilePath] string callerFilePath = "",
+            [CallerLineNumber] int callerLineNumber = 0
         ) => Log.DispatchLog(
             LogLevel.Debug,
             string.Concat(MessagePrefix, message),
-            DerivedTypeName,
-            callerPath,
-            callerName,
-            callerLine);
+            OwnerTypeName,
+            callerMethodName,
+            callerFilePath,
+            callerLineNumber);
```

Comment:
I hesitated to reorganize these, but the consistency is nice to have, and I assume there aren't a lot of cases outside Anvil that were supplying custom values for these args, or few enough they can be easily updated.

Reply:
yea, all good. Just need to be super careful for `UnityLogListener` and the like that manually supply these values